src/index.js

```diff
@@ -15,6 +15,25 @@ const validateFunctionOrUndefined = (param, paramName) => {
   if (!(typeof param === 'undefined' || typeof param === 'function')) throw new Error(`Please provide correct value for ${paramName}`)
 }
 
+const handleBulkResponseErrors = (bulkResponse, body) => {
+  if (bulkResponse.errors) {
+    const erroredDocuments = []
+    bulkResponse.items.forEach((action, i) => {
+      const operation = Object.keys(action)[0]
+      if (action[operation].error) {
+        erroredDocuments.push({
+          status: action[operation].status,
+          error: action[operation].error,
+          operation: body[i * 2],
+          document: body[i * 2 + 1]
+        })
+      }
+    })
+    console.error(erroredDocuments)
+    throw new Error(erroredDocuments.map(obj => obj.error.reason).join(','))
```

Comment:
Why not throw with the object created above (erroredDocuments)?

Reply:
Kind of same as above I guess, we use this library on many lambdas and if we do that we don't get enough meaningful details in cloudwatch. But if DEBUG=* works then I am happy to remove it / change it.